docs/api.yaml

```diff
@@ -607,4 +607,4 @@ servers:
   - url: 'https://localhost'
     description: 'Local testing'
   - url: 'https://inference.cloud.ai4eosc.eu'
-    description: 'AI4EOSC's OSCAR cluster'
+    description: 'AI4EOSCs OSCAR cluster'
```

Comment:
This change should be reverted as per https://www.ef.co.uk/english-resources/english-grammar/forming-possessive/